composer.json

```diff
@@ -65,6 +66,10 @@
         "symfony/css-selector": "^4.3",
         "symfony/phpunit-bridge": "^5.0"
     },
+    "suggest": {
+        "sonata-project/notification-bundle": "^2.2",
```

Comment:
The suggest section is not for the version

Reply:
This should not be in suggest. My mistake while copy it from other truncate PR.